### PR TITLE
Fix container regex

### PR DESCRIPTION
**Pull request by ghost**
There are some small edge cases in some container images which have tags with a version number that has a dot, but there is no latest tag or one without dots.

This PR allows those images to be used.

It also adds a test to go along with the fix.

And this also fixes dashes in the tag name.

### CODE REVIEW

File: frontend/coprs_frontend/tests/test_forms.py [hunk 1]
```diff
@@ -108,6 +108,7 @@ def test_form_regexes():
     assert re.match(REGEX_BOOTSTRAP_IMAGE, "fedora")
     assert re.match(REGEX_BOOTSTRAP_IMAGE, "registry.fedoraproject.org/fedora:rawhide")
     assert re.match(REGEX_BOOTSTRAP_IMAGE, "registry.fedoraproject.org/fedora")
+    assert re.match(REGEX_BOOTSTRAP_IMAGE, "example.com/test:2.0")
```
[the remark this answers is not in the document]
Comment (FrostyX) [reply frontend/coprs_frontend/tests/test_forms.py:111]: Thank you for the PR @qikp0,
it looks good to me, and I am okay with merging as is.

But I agree with Gemini here. Do you want to add also a test for a tag name that contains a dash? So that we don't accidentally break the support in the future?

[the remark this answers is not in the document]
Comment (praiskup) [reply frontend/coprs_frontend/tests/test_forms.py:111]: Yes, please.

[the remark this answers is not in the document]
Comment (ghost) [reply frontend/coprs_frontend/tests/test_forms.py:111]: @FrostyX @praiskup fixed.

**Comment by praiskup** [comment]
Thank you!
